static/js/shared/data_fetcher.test.ts

```diff
@@ -160,6 +160,89 @@ test("fetch stats data", () => {
   });
 });
 
+test("fetch stats data with per capita", () => {
```

Comment:
please update the test name with the subject of this test (that we're testing per capita with population 0)

you can also simplify this test by doing it for only one state.

Reply:
done